server.js

```diff
@@ -9,7 +14,56 @@ app.get('/api/foods', function(request, response) {
   Food.findAllFoods().then(function(data){
     response.json(data.rows)
   })
+})
+
+app.get('/api/foods/:id', function(request, response) {
+  var id = request.params.id
+
+  Food.find(id).then(function(data){
+    if(data.rowCount == 0) {
+      return response.status(404).send({ error: 'Invalid id, try again.'})
+    }
+    response.json(data.rows[0])
+  })
+})
+
+app.post('/api/foods', function(request, response) {
+  var name = request.body.name
```

Comment:
@case-eee for some reason, all tests pass with `request.body` but Postman does not work to send in params. Postman works with `request.query` but then the tests fail. Any idea what could be causing this? We've spoken with a couple other groups experiencing the same thing, and research has turned up nothing